quickcheck-dynamic/src/Test/QuickCheck/StateModel/Variables.hs

```diff
@@ -37,6 +37,10 @@ import Test.QuickCheck as QC
 newtype Var a = Var Int
   deriving (Eq, Ord, Typeable, Data)
 
+-- | Create a fresh symbolic variable with given identifier. While 'Var's are
```

Comment:
```suggestion
-- | Create a fresh symbolic variable with given index. While 'Var's are
```

Reply:
Index in what?

Reply:
I think `Var`'s "index" can be thought as a "de Bruijn index", eg. an index in the enclosing scopes. It's _not_ a de Bruijn index I think, because such an index would be relative.

Reply:
(You don't need to make that replacement necessarily, but that's a good way to think about it)